states/menu.py

```diff
@@ -5,22 +5,29 @@
 
 
 class MenuOptions(StateOptions):
+    """Enum values for options available in the menu"""
+
     BATTLE = 1
     TEAM = 2
     EXIT = 3
 
 
 class Menu(State):
+    """Displays, holds, and reads options for Menu state change"""
+
     option: Optional[MenuOptions] = None
 
     def __init__(self):
+        """Initializes the menu state and lists available options checking against user input if exit is not chosen"""
```

Comment:
I said to get rid of `__init__()` comments, but this one does more than a standard init so I think it's good to keep. Mark as resolved 👍